interfaces/mount/spec.go

```diff
@@ -206,3 +216,31 @@ func (spec *Specification) AddPermanentSlot(iface interfaces.Interface, slot *sn
 	}
 	return nil
 }
+
+// AddOvername records mappings of snap directories.
+//
+// When the snap is installed with an instance key, set up it's mount namespace
+// such that it appears as a non-instance key snap. This ensures compatibility
```

Comment:
The comment about compatibility is no longer valid, is it? I think that we say $SNAP_DATA is /var/snap/123 in all the cases (regardless of instance keys) precisely because of this mapping. In other words, the comment reflects the old reality where we had an extra set of instance-specific variables. I think that this part can be removed.